tests/console/ansible.pm

```diff
@@ -156,29 +177,28 @@ sub run {
 }
 
 sub cleanup {
-    # Logout $testapi::username
-    enter_cmd 'exit';
-
-    # Make sure that root console is logged off before we reset the consoles
-    select_console 'root-console';
-    enter_cmd 'exit';
-
-    # Make sure that user console is logged off before we reset the consoles
-    select_console 'user-console';
-    enter_cmd 'exit';
-
-    # Reset consoles and log in root
-    reset_consoles;
-    select_serial_terminal;
-
     # Remove all the directories ansible created
     assert_script_run 'rm -rf ~/ansible_collections/ /tmp/ansible/';
 
     # Remove the ansible sudoers file
     assert_script_run 'rm -rf /etc/sudoers.d/ansible';
 
+    # Remove the ansihle_rsa key
+    assert_script_run 'rm -rf ~/.ssh/ansible_rsa*';
+
+    # Remove the johnd user created in the ansible playbook
+    assert_script_run 'userdel -rf johnd';
+
     # Remove ansible, yamllint and git
-    zypper_call 'rm ansible git-core python3-yamllint ed';
+    if (is_transactional) {
+        select_console 'root-console';
+        trup_call("pkg remove $pkgs");
```

Comment:
That results in 'sudo' being uninstalled:
https://openqa.opensuse.org/tests/2838172#step/ansible/175

which is likely the reason for the following `toolbox -r id` not working anymore (-r = --root)

Reply:
#15794